package.json

```diff
@@ -46,7 +46,7 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test",
+    "test": "jest",
```

Comment:
this should return to react-scripts test please. just run npm test to get the auto test runner to activate. you can run with watch mode so the tests run on every file change.

Reply:
Yes, your way is much convenient, thanks for letting me know!